src/python/pants/reporting/zipkin_reporter.py

```diff
@@ -40,12 +41,14 @@ class ZipkinReporter(Reporter):
   """Reporter that implements Zipkin tracing .
   """
 
-  def __init__(self, run_tracker, settings, endpoint):
+  def __init__(self, run_tracker, settings, endpoint, trace_id, parent_id):
```

Comment:
Let's add some pydoc here to explain what these parameters all are, now that we have some that are a little less obvious :)

Reply:
We actually add pydoc for methods as the first line _inside_ the method, rather than just before :) (Why would languages be consistent!?)